packages/@o3r/rules-engine/src/stores/rulesets/rulesets.selectors.ts

```diff
@@ -39,7 +39,7 @@ export const selectActiveRuleSets = createSelector(
   selectAllRulesets,
   (ruleSets) => ruleSets.filter((ruleSet: Ruleset) => {
 
-    if (ruleSet.linkedComponent) {
+    if (ruleSet.linkedComponents?.or?.length || ruleSet.linkedComponent) {
```

Comment:
why should we return false if there are linkedComponents?
I know this was the behaviour before, but I fail to understand why we would only consider only ruleset applied without this condition and call this selector "selectActiveRulesets"

Reply:
ruleset which have defined linkedComponents ar not considered active by default. 
That's why they are not considered in this selector and they have their own selector. You are right about the naming. It should be 'selectDefaultRulesets' or 'selectDefaultActiveRulesets'. 
Do you think I should create a second one and deprecate the existing one ?